qiskit_nature/circuit/library/primitives/bogoliubov_transform.py

```diff
@@ -0,0 +1,265 @@
+    and the :math:`\{b_j\}` are the new fermionic creation operators.
+    The new creation operators are linear combinations of the original ladder operators,
+    and the coefficients of the linear combinations are specified by a matrix :math:`W`
+    which determines the unitary :math:`U`. The matrix :math:`W` is either
+    :math:`N \times N` or :math:`N \times 2N`.
+
+    If :math:`W` is :math:`N \times N`, then the linear combinations involve only the
+    original creation operators:
+
+    .. math::
+        \begin{pmatrix}
+            b^\dagger_1 \\
+            \vdots \\
+            b^\dagger_N \\
+        \end{pmatrix}
+        = W
+        \begin{pmatrix}
+            a^\dagger_1 \\
+            \vdots \\
+            a^\dagger_N \\
+        \end{pmatrix}.
+
+    If :math:`W` is :math:`N \times 2N`, then the linear combinations involve both the
+    original creation and annihilation operators:
+
+    .. math::
+        \begin{pmatrix}
+            b^\dagger_1 \\
+            \vdots \\
+            b^\dagger_N \\
+        \end{pmatrix}
+        = W
+        \begin{pmatrix}
+            a^\dagger_1 \\
+            \vdots \\
+            a^\dagger_N \\
+            a_1 \\
+            \vdots \\
+            a_N
+        \end{pmatrix}.
+
+    The matrix :math:`W` is commonly obtained by calling the
+    :meth:`~.QuadraticHamiltonian.diagonalizing_bogoliubov_transform`
+    method of the :class:`~.QuadraticHamiltonian` class.
+
+    Currently, only the Jordan-Wigner Transformation is supported.
+
+    References:
+        - `arXiv:1711.05395`_
+        - `arXiv:1603.08788`_
+
+    .. _arXiv:1711.05395: https://arxiv.org/abs/1711.05395
+    .. _arXiv:1603.08788: https://arxiv.org/abs/1603.08788
+    """
+
+    def __init__(
+        self,
+        transformation_matrix: np.ndarray,
+        qubit_converter: Optional[QubitConverter] = None,
+        validate: bool = True,
+        rtol: float = 1e-5,
+        atol: float = 1e-8,
+        **circuit_kwargs,
+    ) -> None:
+        r"""
+        Args:
+            transformation_matrix: The matrix :math:`W` that specifies the coefficients of the
+                new creation operators in terms of the original creation operators.
+                Should be either :math:`N \times N` or :math:`N \times 2N`.
+            qubit_converter: A QubitConverter instance.
+            validate: Whether to validate the inputs.
+            rtol: Relative numerical tolerance for input validation.
+            atol: Absolute numerical tolerance for input validation.
+            circuit_kwargs: Keyword arguments to pass to the QuantumCircuit initializer.
+
+        Raises:
+            ValueError: transformation_matrix must be a 2-dimensional array.
+            ValueError: transformation_matrix must have orthonormal rows.
+            ValueError: transformation_matrix does not describe a valid transformation
+                of fermionic ladder operators
```

Comment:
Similar comment to above - and I think its the same constraint right though the docstring of the parameter here does not say similar

Reply:
Done.